packages/common/telemetry.ts

```diff
@@ -1,7 +1,12 @@
 import { post } from './fetchWrappers'
 
 export function handlePageTelemetry(API_URL: string, route: string, telemetryProps: any) {
```

Comment:
I reckon it's best we define the shape of telemetryProps here - rather than leaving it as `any`

at least TS can help us check for issues